Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -99,6 +69,7 @@ public enum Route: DerivePartialIsos {
 
   public enum Feed: DerivePartialIsos {
     case atom
+    case episodes
```

Comment:
this is an RSS/atom feed of our episodes that is podcast/itunes friendly. we might be able to just merge this with the existing atom feed so that we dont have to support both. can chat more IRL about what that would mean and what it would take to do...